lesson7-questions.txt

```diff
@@ -87,19 +88,19 @@ You will see the SQL that is executed logged to the screen.  What is that SQL?
 There is a security exposure, a risk that a hacker could do damage.  What is that
 risk? What two things are done in your code to prevent the risk?
 
-
+check_logon and check_access make it so a user can only delete their own posts. 
 
 9. The routes for subscriptions are nested under the forum routes.  How does this
 change the routes?  What is the additional hash key and value that is passed in the
 params hash to the subscriptions controller as a result of this nesting? Which routes
 are unaffected by the nesting?
 
-
+subscriptions#create and #new both have /forums/:forum_id in the routes. the params might be forum_id: *a forum id number* but I intend to come back to this question once I get the subscriptions working this week
```

Comment:
you've pretty much got it -- but I always encourage coming back to review things. :) something to think about is why would we nest it this way by design?